sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkMeterProvider.java

```diff
@@ -42,9 +42,9 @@
   private final MeterProviderSharedState sharedState;
 
   SdkMeterProvider(
-      Clock clock, Resource resource, ViewRegistry viewRegistry, ExemplarSampler exemplarSampler) {
+      Clock clock, Resource resource, ViewRegistry viewRegistry, ExemplarFilter exemplarFilter) {
```

Comment:
Cool, I like Filter a lot better since we apply reservoir sampling after it anyways. Perhaps it allows renaming what we have as ExemplarReservoir to ExemplarFilter :-D